src/fileio/fileio_party.cpp

```diff
@@ -209,11 +210,11 @@ bool load_party_v1(fs::path file_to_load, cUniverse& real_univ, bool town_restor
 			fin.read((char*)&o_maps, len);
 		}
 		
-		// LOAD SFX & MISC_I
+		// SKIP SFX & MISC_I
```

Comment:
Why skip them?

Reply:
This is the commit: 441ebb358f9312be01cbbaa286167d5bc69adf7d

Reply:
Yes, I made a mistake by skipping SFX & MISC_I, i.e. I probably thought the old code called `load_town` with mode=0 (but it only calls it with mode=1 and mode =2) => I will undo that.

The other patches are basic cleanups and fix some errors: import of special_item, check that there are no duplicate creatures for the same town, add some forgotten flip_short, use out_e instead of expl (because out_e replaced expl), ...